### PR TITLE
Add new column template for displayLeft/Right hook

### LINKED ISSUE #20100: Link list module in left/right column
#### Is your feature request related to a problem?

When transplanting the link widget to left or right column (for example on cms page), the blocks are created as columns inside the left/right column.

image

#### Describe the solution you'd like

In left/right column, the different link blocks should just be above eacht other, in a single column.
Maybe a different template file depending on the hook the module is in?

#### Additional context

Working in PS 1.7.6.5

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Link list was broken on displayLeftColumn and displayRightColumn hook
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #20100.
| How to test?  | See issue (should probably be tested with https://github.com/PrestaShop/ps_linklist/pull/104)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22076)

### CODE REVIEW

**Review by Arman-Hosseini — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

File: themes/classic/modules/ps_linklist/views/templates/hook/linkblock-column.tpl [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ *}
+<div class="links">
+  {foreach $linkBlocks as $linkBlock}
+    <div class="wrapper">
+      <p class="h3 hidden-sm-down">{$linkBlock.title}</p>
+      {assign var=_expand_id value=10|mt_rand:100000}
+      <div class="title clearfix hidden-md-up" data-target="#footer_sub_menu_{$_expand_id}" data-toggle="collapse">
```
Comment (PululuK) [inline themes/classic/modules/ps_linklist/views/templates/hook/linkblock-column.tpl]: You can use   `$linkBlock.id`  instead of random value?

Comment (NeOMakinG) [reply themes/classic/modules/ps_linklist/views/templates/hook/linkblock-column.tpl]: Copy pasted the basic template and removed bootstrap classes, I think if we wants to improve the module, let's do it on another PR with another issue :)

Comment (PululuK) [reply themes/classic/modules/ps_linklist/views/templates/hook/linkblock-column.tpl]: As I understand it, here this is an override, so I think we can fix that here.

**Review by atomiix — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by Roomsmaniia** [comment]
LGTM,

pr
pr2

test with : https://github.com/PrestaShop/ps_linklist/pull/104 

Thanks @NeOMakinG

**Comment by PululuK** [comment]
Thanks @NeOMakinG

**Pull request merged**